lib/session.js

```diff
@@ -316,12 +327,48 @@ const selectCurrentSession = async function(account) {
   }
 };
 
+const createEconomischeActorByClaims = async function(claims) {
```

Comment:
There is a risk here we end up with double URI's. Just be aware we need a strategy to mitigate is.
Also; this kind of query somewhat hints towards a separate new service.